pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
     "platformdirs>=4.3.6",
     'typing-extensions>=4.10.0; python_version<"3.13"',
     "scikit-video>=1.1.11",
+    "scipy>=1.13.0",
```

Comment:
made a lockfile that splits out <3.10 and >=3.10, noticed that we are using scipy within the package but don't have it as a direct dependency